doc/nrf/external_comp/bt_fast_pair.rst

```diff
@@ -172,6 +172,10 @@ To support the FMDN extension, set the **Find My Device** feature to **true** fo
 
 For an example that uses the **Find My Device** feature, see the :ref:`fast_pair_locator_tag` sample.
 
+.. note::
+   To be able to test the FMDN extension with debug (uncertified) device models, you need to set up your Android test device with the primary email account that is part of the Google email allow list for the FMDN feature.
```

Comment:
I would suggest to simplify a bit:
`If you want to test FMDN extension with a debug Model ID, primary email account used by your Android phone must be allowlisted by Google.`?